plugins/lyrics-genius/back.js

```diff
@@ -7,20 +7,46 @@ const fetch = require("node-fetch");
 
 const { cleanupName } = require("../../providers/song-info");
 const { injectCSS } = require("../utils");
+var revRomanized = false; 
 
-module.exports = async (win) => {
+module.exports = async (win, options) => {
+	if(options.romanizedLyrics) {
+		revRomanized = true;
+	}
 	injectCSS(win.webContents, join(__dirname, "style.css"));
 
 	ipcMain.on("search-genius-lyrics", async (event, extractedSongInfo) => {
+		console.log("Fetching Lyrics");
 		const metadata = JSON.parse(extractedSongInfo);
 		event.returnValue = await fetchFromGenius(metadata);
 	});
 };
 
+const toggleRomanized = () => {
+	revRomanized = !revRomanized;
+	console.log("Romanized Mode: " + revRomanized);
+}
+
 const fetchFromGenius = async (metadata) => {
-	const queryString = `${cleanupName(metadata.artist)} ${cleanupName(
-		metadata.title
-	)}`;
+	/* Tried using regex to test the title and artist for East Asian Characters. It works but I realized 
+	some groups are fully English in both title and singer. Might come back to this method in the future.
+	*/
+	const songTitle = `${cleanupName(metadata.title)}`;
+	const songArtist = `${cleanupName(metadata.artist)}`;
+	// let regexEastAsianChars = new RegExp("[\u{3040}-\u{30ff}\u{3400}-\u{4dbf}\u{4e00}-\u{9fff}\u{f900}-\u{faff}\u{ff66}-\u{ff9f}]");
+	// let hasAsianChars = regexEastAsianChars.test(songTitle) || regexEastAsianChars.test(songArtist);
+	// console.log(songTitle);
+	// console.log(songArtist);
+
+	const queryString = revRomanized ? 
+	`${songTitle}`.concat(" Romanized") : 
+	`${songArtist} ${songTitle}`;
+
+	// const queryString = `${cleanupName(metadata.artist)} ${cleanupName(
+	// 	metadata.title
+	// )}`;
+	console.log(queryString);
+
```

Comment:
Since you decided against it (rightfully IMO)

please remove all those code comments

```suggestion
	const songTitle = `${cleanupName(metadata.title)}`;
	const songArtist = `${cleanupName(metadata.artist)}`;

	const queryString = revRomanized ? 
	`${songTitle}`.concat(" Romanized") : 
	`${songArtist} ${songTitle}`;

```